config/environments/development.rb

```diff
@@ -56,4 +56,6 @@
 
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+
+  config.active_storage.service = :local
```

Comment:
Ah, so I need to document this better. I mentioned it in a message in `storage.yaml`, but basically we are using ActionText, Rails's built-in system for storing rich text (i.e. text with formatting) in databases, and ActionText has an optional ability to embed images in it, but because of that it REQUIRES you to have ActiveStorage, Rails's built-in system for storing files, set up. We don't _need_ ActiveStorage, but we have to have it configured, so this says: "If we ever did upload an image, save it as a file on this server (rather than trying to upload it to AWS S3 or some other dedicated image storage service)".